packages/test-interface-criteria/src/criteria/TestOpportunityBookableOutsideValidFromBeforeStartDate.js

```diff
@@ -11,13 +11,15 @@ const { InternalCriteriaFutureScheduledAndDoesNotRequireDetails } = require('./i
  */
 function mustHaveBookingWindowAndBeOutsideOfIt(offer, opportunity, options) {
   const dateAfterWhichBookingsCanBeMade = getDateAfterWhichBookingsCanBeMade(offer, opportunity);
-  if (dateAfterWhichBookingsCanBeMade == null) {
+  const dateBeforeWhichBookingsCanBeMade = getDateBeforeWhichBookingsCanBeMade(offer, opportunity);
+  if (dateAfterWhichBookingsCanBeMade == null && dateBeforeWhichBookingsCanBeMade == null) {
     return false; // has no booking window
   }
   /* If, within 2 hours, the booking window would be reached, it may be possible for this to happen
   during the test run. So, to be on the safe side, we only accept Opportunities whose booking window
   starts at least 2 hours in the future. */
-  return options.harvestStartTimeTwoHoursLater < dateAfterWhichBookingsCanBeMade;
+  return (dateAfterWhichBookingsCanBeMade == null || options.harvestStartTimeTwoHoursLater < dateAfterWhichBookingsCanBeMade)
+    || (dateBeforeWhichBookingsCanBeMade == null || options.harvestStartTime > dateBeforeWhichBookingsCanBeMade);
```

Comment:
```suggestion
    || (dateBeforeWhichBookingsCanBeMade == null || options.harvestStartTimeTwoHoursLater > dateBeforeWhichBookingsCanBeMade);
```

Want to be outside the 2 hour window right? The criteria below indicates that too. But timezones mess with my head so I could have read this wrong.

Reply:
I think this is right as is. If it was outside the "2 hours later" time, then this criteria could falsely identify an event that is bookable until 1h30-after-harvest-start as not-bookable-due-to-being-outside-of-the-booking-window. Then, if an attempt is made to book the event 30mins into harvest start, it would succeed, which goes against the criteria

Reply:
Agree, it looks right as is - good to check though!